web/firefoxcom.js

```diff
@@ -314,6 +314,12 @@ class FirefoxScripting {
   }
 }
 
+class MLManager {
+  guess(data) {
+    return FirefoxCom.requestAsync("guess", data);
```

Comment:
Similar to a previous comment, given the "general" nature of this name, how about the following?
```suggestion
    return FirefoxCom.requestAsync("mlGuess", data);
```